addons/binding/org.openhab.binding.allplay/src/main/java/org/openhab/binding/allplay/internal/AllPlayBindingProperties.java

```diff
@@ -17,25 +17,34 @@
  */
 public class AllPlayBindingProperties {
 
-    private final Integer rewindSkipTimeInSec;
-    private final Integer fastForwardSkipTimeInSec;
+    private final double rewindSkipTimeInSec;
```

Comment:
Do you really need doubles here? Or could you simply change the parameter declaration to be Integer (in the binding.xml)?